src/main/resources/db/changelog/changes/db.changelog-content.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="Maciej" id="create_contents_table">
+        <dropTable  cascadeConstraints="true"
+                    tableName="contents"/>
+        <createTable tableName="contents">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="page" type="VARCHAR(255)">
+                    <constraints nullable="true"/>
+            </column>
+            <column name="key" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="value_pl" type="VARCHAR(255)">
```

Comment:
Are u sure inputs will be less than 255 chars? @Pauliiiba @BarSwi u should know it. If more use TEXT

Reply:
No idea, but there is a big chance it won't be enough. You are right, I will set TEXT